now.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "oneGlance",
```

Comment:
I like my app title as well, but i encourage you to type in your own app title ;)

Reply:
Have you already a google alert running? 👍

Reply:
My ears are ringing if someone violates my copyright. 
My lawyer will contact you soon!!!